sirepo/template/cloudmc.py

```diff
@@ -469,24 +469,24 @@ def _generate_distribution(dist):
     return _generate_call(t, args)
 
 
-def _generate_materials(data):
+def _generate_materials(data, v):
```

Comment:
Use a descriptive variable name (j2_ctx?) when it is a function parameter (here and in the other methods). This will also mean you don't need to rename d to v.